Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -5,11 +5,11 @@
 import StreamChat
 import UIKit
 
-/// A `ChatChannelSwipeableListItemView` subclass view that shows channel information.
+/// An ItemView which is embed inside `ChatChannelListCollectionViewCell` and displays content about channel.
 public typealias ChatChannelListItemView = _ChatChannelListItemView<NoExtraData>
```

Comment:
Again - the documentation is a bit confusing:
```
An ItemView which is embed inside `ChatChannelListCollectionViewCell`
```
It sounds like the view is embedded inside the cell, but that's not true. This is a normal `UIView` that can be perfectly used without the cell. too.

What about:
```
A `UIView` subclass that shows summary and preview information about a given channel.
```

Reply:
Okay, wanted to say that it's embed in cell... But yeah it can be embed anywhere... Changed :)